### ISSUE TITLE
Multiline Texts

### ISSUE THREAD

**Issue by BafDyce** [bevyengine/bevy#182]
Apparently `Text` [(docs)](https://docs.rs/bevy/0.1.2/bevy/prelude/struct.Text.html) does not support multi line strings. Not sure if this is expected (but undocumented) behavior or a bug?

I simply made the following change to the `examples/ui/text.rs` example:

```diff
diff --git a/examples/ui/text.rs b/examples/ui/text.rs
index ecd7f081..c511ef38 100644
--- a/examples/ui/text.rs
+++ b/examples/ui/text.rs
@@ -17,7 +17,7 @@ fn text_update_system(diagnostics: Res<Diagnostics>, mut query: Query<&mut Text>
     for mut text in &mut query.iter() {
         if let Some(fps) = diagnostics.get(FrameTimeDiagnosticsPlugin::FPS) {
             if let Some(average) = fps.average() {
-                text.value = format!("FPS: {:.2}", average);
+                text.value = format!("FPS: {:.2}\nsecond line", average);
             }
         }
     }
```

**Expected behavior**: two lines of output

**Actual behavior**: single line of output:
![image](https://user-images.githubusercontent.com/9869387/90243568-6e163600-de2f-11ea-82c0-c3b8423fd96a.png)

**Comment by cart** [comment]
Yeah this is an expected behavior. We just haven't implemented line breaking yet. Thanks for the pr!

**Comment by Moxinilian** [comment]
Fixed by #183 and #765
